src/layouts/Settings/AnalyticsSettings.tsx

```diff
@@ -34,8 +34,23 @@ export const AnalyticsSettings = ({
         </FormControl>
 
         <FormControl isDisabled={isError}>
-          <FormLabel>Google Analytics</FormLabel>
-          <Input w="100%" {...register("ga")} />
+          <FormLabel>Google Analytics (UA)</FormLabel>
+          <SectionCaption label="" icon={BiInfoCircle}>
```

Comment:
do we need the empty label?

Reply:
Our `SectionCaption` component expects a label param - I think we could make the label param optional, have changed it in https://github.com/isomerpages/isomercms-frontend/pull/1229/commits/5167464019bf18b28c1c64eaad1a337c327b8e9b